tests/lib/rules/require-default-props.js

```diff
@@ -211,6 +211,32 @@ ruleTester.run('require-default-props', rule, {
       `,
       options: [{ ignoreFunctionalComponents: true }],
     },
+    {
+      code: `
+        function Hello(props) {
+          return <div>Hello {props.foo}</div>;
+        }
+        Hello.propTypes = {
+          foo: PropTypes.string
+        };
+        Hello.defaultProps = {
+          foo: 'bar;'
+        }
+      `,
+      options: [{ forbidDefaultForRequired: true }],
```

Comment:
Shouldn’t this be invalid, since foo is required but has a defaultProp?

Reply:
> ```ts
> Hello.propTypes = {
>   foo: PropTypes.string
> };
> ```

There is no `.isRequired`. Am I missing something?

Reply:
oh, no, you’re right :-)